server/src/conf/logback.xml

```diff
@@ -29,9 +29,17 @@
         </encoder>
     </appender>
 
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <charset>UTF-8</charset>
+            <Pattern>%d %-4relative [%thread] %-5level %logger{35} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
     <root level="${log.level:-INFO}">
         <appender-ref ref="FILE"/>
+        <appender-ref ref="consoleAppender"/>
```

Comment:
I think the ref needs to match the name attribute of the appender?  (STDOUT) in this case.

Unless there's a defaulting mechanism based on the class name or something.